spec/controllers/jasmine_fixtures/streams_spec.rb

```diff
@@ -21,7 +21,7 @@
           Timecop.travel(time += 1.minute)
           posts << alice.post(:status_message, :text => "hella infos yo!", :to => alice.aspects.first.id)
           Timecop.travel(time += 1.minute)
-          posts << alice.post(:reshare, :root_guid => FactoryGirl.create(:status_message, :public => true).guid, :to => 'all')
+          posts << alice.post(:reshare, :root_guid => FactoryBot.create(:status_message, :public => true).guid, :to => 'all')
```

Comment:
- Style/HashSyntax: Use the new Ruby 1.9 hash syntax.
- Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.
- Layout/LineLength: Line is too long. [125/120]